### PR TITLE
whitespace (remove TAB usage)

### PR DESCRIPTION
**Pull request by gunnarx**
tl;dr... There were some random TAB characters instead of 8 spaces, so I fixed as many files I could find.   Mostly source files .c/.h, a few shell scripts.

----
...

Perhaps some auto-indenter could have taken care of it (if set to the weird&wonky GNU-style first ;-) 

Anyway I also fixed a couple of inconsistent shell scripts.  Looks like 4-spaces is the norm there.

tests/libtest.sh had some spurious tabs and the two if-statements at the top were just modified from 2 to 4 to be consistent with the rest.

tests/test-configure had one probably undesired tab, but ignored since the rest of that script is generating a Makefile so those parts must use the tabs.  

I ignored everything configure/make related .mk, .inc, .m4 etc.

bwrap/bwrap.xml also has a few spurious tabs, but that needs to be fixed in the submodule.

### CODE REVIEW

**Comment by mwleeds** [comment]
bot, test pull request

**Comment by mwleeds** [comment]
Thanks!

@rh-atomic-bot r+
